file-attribute-caching/src/main/kotlin/com/pkware/filesystem/attributecaching/AttributeCachingPath.kt

```diff
@@ -47,13 +49,28 @@ internal class AttributeCachingPath(
         if (views.contains("posix")) provider.readAttributes(delegate, PosixFileAttributes::class.java) else null
     }
 
+    /**
+     * The cached [AclFileAttributeView] for this [AttributeCachingPath].
+     *
+     * Will be `null` if the [delegate]'s [FileSystemProvider] does not support it.
+     *
+     * Once the [AclFileAttributeView] is cached it can only be modified/re-cached via [Files.setAttribute] or other
+     * attribute setting methods in [[FileSystemProvider].
+     */
+    var aclFileAttributeView by LazyAttribute {
```

Comment:
If my other suggestion is followed, this could just be `private var`.

Reply:
See above replies